plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -71,12 +71,26 @@ class RayNodeConf(Dict[str, Any]):
     ImageId: str = MISSING
 
 
+@dataclass
+class RayCodeSyncConf:
+    """
+    If your code contains more than one modules/files, you need to sync your code to the remote cluster.
+    The code will be synced to a tmp dir on the cluster.
+    code_dir could be a relative dir to where you run your app or a absolute dir.
+    Leave this empty if you don't need to sync code.
+    """
```

Comment:
A few comments:
>code_dir could be a relative dir to where you run your app or a absolute dir.
relative to what? cwd?

for empty use None, not "".
you can pass None from yaml with the literal null.